Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -1052,6 +1052,22 @@ bool gstObjectHasProperty(GstPad* pad, const char* name)
     return gstObjectHasProperty(GST_OBJECT_CAST(pad), name);
 }
 
+gboolean webKitGstElementSetBandwidthLimitedQuery(GstElement* element, GstQuery* query)
+{
+    auto klass = G_OBJECT_GET_CLASS(element);
+    gboolean result = GST_ELEMENT_CLASS(klass)->query(element, query);
```

Comment:
This is wrong actually, triggers infinite recursion. I'm not sure how to do this properly, for now I'll duplicate this method in both MSE and mediastream elements.